src/test/java/models/testUser.java

```diff
@@ -30,9 +30,11 @@ public void testUserHand() {
     public void testTakeCard() {
         //Arrange
         User u = new User();
+        Deck deck = new Deck();
 
         //Act
-        u.takeCard();
+        deck.build();
+        u.takeCard(deck.deal());
 
```

Comment:
If you are passing a card from the deck to the user hand the hand count should not be zero, it should be one.